extensions/github-stars/package.json

```
@@ -0,0 +1,55 @@
+{
+  "name": "search-github-stars",
+  "title": "GitHub Stars",
+  "description": "Display and filter your recent GitHub stars",
+  "subtitle": "GitHub",
+  "icon": "icon.png",
+  "author": "mrmartineau",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Stars",
```

```suggestion
      "title": "Starred Repositories",
```

This sounds a lot more natural